charts/internal/shoot-control-plane/templates/metallb.yaml

```diff
@@ -437,6 +437,16 @@ spec:
       # https://github.com/metallb/metallb/issues/462
       - name: health
         image: {{ index .Values.images "kubectl" }}
+        resources:
+          limits:
+            cpu: 100m
+            memory: 100Mi
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - all
+          readOnlyRootFilesystem: true
```

Comment:
this breaks caching for kubectl
`kubectl create --save-config configmap health --dry-run=client -o yaml --from-literal=configStale=${stale} --from-literal=configLoaded=${loaded} | kubectl --cache-dir=/tmp/.kube/.cache/ apply -f -`

```
NAMESPACE                               LAST SEEN   TYPE      REASON         OBJECT                                               MESSAGE
default                                 44m         Warning   OOMKilling     node/shoot--node-group-0-7df4d-dhf7w   Memory cgroup out of memory: Killed process 1510720 (kubectl) total-vm:828180kB, anon-rss:99844kB, file-rss:28092kB, shmem-rss:0kB, UID:65534 pgtables:368kB oom_score_adj:-997
default                                 31m         Warning   OOMKilling     node/shoot--node-group-0-7df4d-dhf7w   Memory cgroup out of memory: Killed process 1527065 (kubectl) total-vm:831220kB, anon-rss:99572kB, file-rss:30012kB, shmem-rss:0kB, UID:65534 pgtables:376kB oom_score_adj:-997
default                                 15m         Warning   OOMKilling     node/shoot--node-group-0-7df4d-dhf7w   Memory cgroup out of memory: Killed process 1547965 (kubectl) total-vm:829268kB, anon-rss:99688kB, file-rss:27324kB, shmem-rss:0kB, UID:65534 pgtables:368kB oom_score_adj:-997
default                                 3m19s       Warning   OOMKilling     node/shoot--node-group-0-7df4d-dhf7w   Memory cgroup out of memory: Killed process 1563434 (kubectl) total-vm:828436kB, anon-rss:99256kB, file-rss:26036kB, shmem-rss:0kB, UID:65534 pgtables:368kB oom_score_adj:-997
```

Reply:
Maybe the ro filesystem was too much. :(
@chbmuc Can we revert the filesystem restriction?

Reply:
sure - I've created #280